### PR TITLE
Fix benchmarks spawning detachted tasks. (They where dropped without .detach())

### PR DESCRIPTION
**Pull request by stsydow**
Fix benchmarks spawning detachted tasks. 
They where dropped without .detach() and the execution hangs.

### CODE REVIEW

**Comment by Bathtor** [comment]
Excellent point, I did forget to update those. Thanks for the PR!
